src/main/java/com/aws/greengrass/util/EncryptionUtils.java

```diff
@@ -24,6 +24,9 @@
 
 public final class EncryptionUtils {
 
+    public static final String CERTIFICATE_PEM_HEADER = "-----BEGIN CERTIFICATE-----";
+    public static final String CERTIFICATE_PEM_FOOTER = "-----END CERTIFICATE-----";
```

Comment:
why is the footer not used?

Reply:
The header is all that's needed to split and dedupe the root CA file. I included the footer for completeness even though it's not currently used.